Resume.html

```diff
@@ -0,0 +1,37 @@
+<html>
+	<head>
+		<title> Resume </title>
+
+		<link rel='stylesheet' href="resumestyle.css">
+
+	</head>
+		<Body style='background-image:url(/wheat.jpg)'>
+			<h1> Resume</h1>
+			<table>
+				<th class='Header'> Education</th>
```

Comment:
Something you could do to simplify this just a bit: instead of giving a class to all the `<th>` elements in this table, give a class to the table itself (e.g. "resumeTable"), and then do this in css:
```
.resumeTable th{
   /*put your style here*/
}
```
That CSS is saying "find all the <th> tags inside the resumeTable.